tests/unit/src/test/scala/tests/MtagsEnrichmentsSuite.scala

```diff
@@ -0,0 +1,89 @@
+package tests
+
+import org.eclipse.{lsp4j => l}
+import scala.meta.internal.{semanticdb => s}
+import scala.meta.internal.mtags.MtagsEnrichments._
+
+object MtagsEnrichmentsSuite extends BaseSuite {
+
+  test("XtensionLspRange.encloses - single line") {
```

Comment:
I may be wrong on this, but doesn't the defining the test name like this then not allow the user to target a single test with `testOnly`? For example I tried `unit/testOnly tests.MtagsEnrichmentsSuite."XtensionLspRange.encloses - single line"`, which won't work. Is there a way to do this with a space? Either way, it might be easier to follow the convention of the other tests with shorter dash seperated names in order to more easily target them.

Reply:
Ah, I wasn't aware of the naming convention. I've renamed the tests.